source/spirv_target_env.cpp

```diff
@@ -220,3 +220,31 @@ bool spvIsOpenCLEnv(spv_target_env env) {
   }
   return false;
 }
+
+bool spvIsWebGPUEnv(spv_target_env env) {
+  switch (env) {
```

Comment:
Couldn't we just simplify this to be:

return env == SPV_ENV_WEBGPU_0; ?

we may need to extend it later if we get more then just _0, but that seems like the simplest thing to do at this point?

Reply:
I effectively responded to this on another comment :-/

From the other comment:
I think the advantage of not just iterating the positive cases, is that it makes it easier for a dev to scan for the enum they care about. A find query will hit the enum regardless of it being a positive or negative case, as opposed to having to figure out it is caught by the final return.